hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -89,6 +146,29 @@ const orderSlice = createSlice({
                 payload?.message ||
                 "There was a problem retrieving orders. If this continues please contact hackathon organizers.";
         });
+
+        builder.addCase(cancelOrderThunk.pending, (state) => {
+            state.cancelOrderLoading = true;
+            state.cancelOrderError = null;
+        });
+
+        builder.addCase(cancelOrderThunk.fulfilled, (state, { payload }) => {
+            state.cancelOrderLoading = false;
+            state.cancelOrderError = null;
+            if (payload) {
+                pendingOrderAdapter.removeOne(state, {
+                    payload: payload.id,
+                    type: payload.status,
+                });
+            }
+        });
+
+        builder.addCase(cancelOrderThunk.rejected, (state, { payload }) => {
+            state.isLoading = false;
+            state.cancelOrderError =
```

Comment:
If ur displaying the snack bar already, you don't need this error state bc it's not being used anywhere else